iOS/rainbow/Controller/Camera/CameraController.swift

```diff
@@ -117,6 +117,24 @@ class CameraController: LuminaViewController {
             savedScoreEntry.objects = nil
             try ScoreEntry.ClientPersistence.save(entry: savedScoreEntry)
             cachedScoreEntry = savedScoreEntry
+            
+            //we don't want to send base64 string
+            savedScoreEntry.avatarImage = nil
+            
+            //update the startDate to the cloud.            
```

Comment:
I could be convinced either way on this - should we show `SVProgressHUD` as we send the initial POST request to the server? Might be a good way to properly handle issues if they come up.